lib/compat/wordpress-6.0/client-assets.php

```diff
@@ -105,6 +105,10 @@ function gutenberg_resolve_assets() {
 add_filter(
 	'block_editor_settings_all',
 	function( $settings ) {
+		if ( function_exists( '_wp_get_iframed_editor_assets' ) ) {
+			return $settings;
+		}
```

Comment:
Let's maybe add a bit of an explanation here? Something like

```suggestion
		// The `__unstableResolvedAssets` are generated by `_wp_get_iframed_editor_assets` for WP >= 6.0.
		if ( function_exists( '_wp_get_iframed_editor_assets' ) ) {
			return $settings;
		}
```